.circleci/config.yml

```diff
@@ -81,4 +108,5 @@ workflows:
           filters:
             branches:
               only:
-              - master
+                - master
+      - update_schema
```

Comment:
Since the update_schema is run daily. Are we configuring the trigger via the UI ?

Reply:
we needn't trigger via UI, this `cron` configuration will trigger it
Please let me know if I misunderstand you or you have a deeper think.

Reply:
Previously since the trigger was missing i assumed you intended to add the trigger via the project settings rather than specifying the trigger in the yaml itself since as per the docs setting the trigger via the project settings seems to be the preferred approach. https://circleci.com/docs/migrate-scheduled-workflows-to-scheduled-pipelines/